core.py

```diff
@@ -24,49 +50,52 @@ class Core():
     def __init__(self, tof_lib):
         """ Constructor """
 
-        # Minimum and maximum theoretical pulse widths. Ignore reversing here
-        # ESC "DB1" midpoint is about 1440
-        # ESC "DB2" midpoint is 1500
-        self.LEFT_MIN = 800
-        self.LEFT_MID = 1300
-        self.LEFT_MAX = 1800
-
-        self.RIGHT_MIN = 800
-        self.RIGHT_MID = 1300
-        self.RIGHT_MAX = 1800
-
-        self.LEFT_AUX_1_MIN = 800
-        self.LEFT_AUX_1_MID = 1300
-        self.LEFT_AUX_1_MAX = 1800
-
-        self.RIGHT_AUX_1_MIN = 800
-        self.RIGHT_AUX_1_MID = 1300
-        self.RIGHT_AUX_1_MAX = 1800
-
-        self.left_servo = servo_control.Servo_Controller(
-            self.LEFT_MIN,
-            self.LEFT_MID,
-            self.LEFT_MAX, True)
-        self.right_servo = servo_control.Servo_Controller(
-            self.RIGHT_MIN,
-            self.RIGHT_MID,
-            self.RIGHT_MAX, False)
-
-        self.left_aux_1_servo = servo_control.Servo_Controller(
-            self.LEFT_AUX_1_MIN,
-            self.LEFT_AUX_1_MID,
-            self.LEFT_AUX_1_MAX, False)
-        self.right_aux_1_servo = servo_control.Servo_Controller(
-            self.RIGHT_AUX_1_MIN,
-            self.RIGHT_AUX_1_MID,
-            self.RIGHT_AUX_1_MAX, False)
-
-        self.left_channel = 1
-        self.right_channel = 2
-
-        # Proximity sensor: roughly cm from closest measurable point
-        # self.tof_left = sensor.Sensor(0, 450, 20, 0)
-        # self.PWMservo = PWM.Servo(pulse_incr_us=1)
+        # Create a list of servo's
+        self.servos = dict()
+        # Add Motor Servo's. NOTE: Left motor esc is reversed.
+        self.servos[ServoEnum.LEFT_MOTOR_ESC] = [
+            servo_control.Servo_Controller(
+                800, 1300, 1800, True  # Min, Mid, Max, Reversed
```

Comment:
suggest using name paramsfor these rather than positional arguments. They're easier to parse and less likely to suffer from rot that accompanying comments.